packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1794,8 +1794,19 @@ class PointerScrollEvent extends PointerSignalEvent with _PointerEventDescriptio
     super.position,
     this.scrollDelta = Offset.zero,
     super.embedderId,
+    this.whenIgnored,
   });
 
+  /// Callback called when this event is [ignore]d.
+  final Function? whenIgnored;
+
+  /// Function to call when this event is ignored by the framework.
+  void ignore() {
+    if (whenIgnored != null) {
+      whenIgnored!();
+    }
```

Comment:
nit: `whenIgnored?.call()`

Reply:
I always try `whenIgnored?()`, and when that doesn't work always fall back to the `if`...

Reply:
I will do this until the day I die, it seems.